onebusaway-android/src/main/java/org/onebusaway/android/tripservice/PollerTask.java

```diff
@@ -165,10 +168,11 @@ private Long checkArrivals(ObaArrivalInfoResponse response, Cursor c) {
                     depart = info.getScheduledArrivalTime();
                 }
 
-                return depart;
+                ArrivalInfo arrivalInfo = new ArrivalInfo(mContext, info, 0l, false);
```

Comment:
After my last comment about refactoring text generation logic to `ArrivalInfo`, we'll definitely need to pass in `response.getCurrentTime()` here.